tsc/meetings/2022-05-10.md

```diff
@@ -0,0 +1,108 @@
+Minutes from 132-nd OpenVDB TSC meeting, May 10th, 2022.
+
+Attendees: *Jeff* L., *Nick* A, *Dan* B., *Ken* M., *Andre* P.
+
+Additional Attendees: Sebastian Gaida, J. T. Nelson (Blender)
+
+Regrets: *Rich* J.
+
+Agenda:
+
+1) Confirm quorum
+2) Secretary
+3) Forum
+7) Next meeting
```

Comment:
4) Multires Proposal 
5) Issue 1374: MIME registration 
6) Website updates 
7) OpenVDB Course 
8) Merging Master into vdb_tool 
9) Issue 1162: Quantization in PNanoVDB 
10) Version 10 
11) Release Manager 
12) Next Meeting

Reply:
Oops!  Meant to go back and fix that...